common/changes/master_2017-02-07-22-15.json

```diff
@@ -0,0 +1,15 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "(AutoDoc utility): Add support for enums with explicit assigned values.",
+      "type": "patch"
```

Comment:
I don't think this requires any changes or patches at all. Can you just remove this file or set to none?